### PR TITLE
Make experiment run on OpenBSD

### PR DESCRIPTION
**Pull request by vext01**
These changes get the warmup experiment running on OpenBSD.

So far only tested without reboots. (Carefully) testing rebooting comes next.

OK?

### CODE REVIEW

File: warmup.krun [hunk 1]
```diff
@@ -1,9 +1,12 @@
 import os
+import sys
 from krun.vm_defs import (PythonVMDef, LuaVMDef, JavaVMDef, GraalVMDef,
 		          PHPVMDef, JRubyTruffleVMDef, V8VMDef, NativeCodeVMDef,
 			  find_internal_jvmci_java_bin)
 from krun import EntryPoint
 
+PLATFORM = sys.platform
```
Comment (ltratt) [inline warmup.krun]: This is pointless -- it just obscures what PLATFORM actually is and makes the latter code harder to understand. Let's kill it.

**Comment by ltratt** [comment]
Apart from the `PLATFORM` over-engineering, this looks good. Let's fix that, then squash anything you want to squash, and we can merge.

**Comment by vext01** [comment]
OK?

**Comment by ltratt** [comment]
Thanks!

**Pull request merged**